### PR TITLE
Follow embulk 0.6.12

### PR DESCRIPTION
**Pull request by uu59**

### CODE REVIEW

File: lib/embulk/input/sfdc.rb [hunk 1]
```diff
@@ -1,8 +1,142 @@
...
+        records.each do |record|
+          values = @schema.collect do |column|
+            SfdcInputPluginUtils.cast(record[column["name"]], column["type"])
+          end
+
+          page_builder.add(values)
+        end
+      end
+
+      # TODO: Replace Embulk::Logger after merging
```
Comment (yoshihara) [inline lib/embulk/input/sfdc.rb]: :eyes: This TODO comment isn't needed?

Comment (uu59) [inline lib/embulk/input/sfdc.rb]: :sweat:

**Comment by yoshihara** [comment]
LGTM :tea:

**Comment by yoshihara** [comment]
Re LGTM :coffee:

**Pull request merged**